plutus-tx-plugin/test/Plugin/Data/monomorphic/irrefutableMatch.plc.golden

```diff
@@ -1,6 +1,9 @@
 (program
   (let
     (nonrec)
+    (datatypebind
```

Comment:
GHC 8.10.7 Core:
```
\ (ds_dieX [Occ=Once1!] :: MyMonoData) ->	
  src<test/Plugin/Data/Spec.hs:97:52-101>	
  case ds_dieX of {	
    __DEFAULT ->	
      patError	
        @ 'LiftedRep @ Integer "test/Plugin/Data/Spec.hs:97:75-100|case"#;
```

GHC 9.2.4 Core:

```
\ (ds_deZU [Occ=Once1!] :: MyMonoData) ->
  src<test/Plugin/Data/Spec.hs:97:52-101>
  case ds_deZU of {
    __DEFAULT ->
      case patError
             @LiftedRep @() "test/Plugin/Data/Spec.hs:97:75-100|case"#
      of {
      };

```

I think this additional definition of `Unit` is the result of the `@()`.